src/NexusMods.App.UI/Controls/Spine/Buttons/ISpineItemViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Windows.Input;
+using NexusMods.App.UI.WorkspaceSystem;
+
+namespace NexusMods.App.UI.Controls.Spine.Buttons;
+
+public interface ISpineItemViewModel : IViewModelInterface
+{
+    /// <summary>
+    /// Is the spine item active and highlighted.
+    /// </summary>
+    bool IsActive { get; set; }
+    
+    /// <summary>
+    /// Command to execute when the spine item is clicked.
+    /// </summary>
+    ICommand Click { get; set; }
```

Comment:
Use `ReactiveCommand` instead of `ICommand`. We used `ICommand` very early on in our codebase without changing the affected types.